src/js/edu-benefits/education-wizard.js

```diff
@@ -41,12 +41,14 @@ function reInitWidget() {
 
   content.addEventListener('change', (e) => {
     const radio = e.target;
-    const otherChoice = radio.dataset.alternate;
-    const otherNextQuestion = container.querySelector(`#${otherChoice}`).dataset.nextQuestion;
-    if (otherNextQuestion) {
-      const otherNextQuestionElement = container.querySelector(`[data-question=${otherNextQuestion}]`);
-      closeStateAndCheckChild(otherNextQuestionElement, container);
-    }
+    const otherChoices = radio.dataset.alternate.split(' ');
```

Comment:
I believe this is governing this part of the wizard

<img width="612" alt="transfered-or-not" src="https://user-images.githubusercontent.com/1969/27942600-1cb23870-62a6-11e7-8b3c-86f2f2799f09.png">

First option is not transferred (not a dependent)
Second option is transferred (a dependent/spouse)
Third option is not transferred (a dependent)

Third option is not considered transferred even though it is using a sponsor’s eligibility. The sponsor was not available to officially transfer the benefit. Does this help?